### ISSUE TITLE
icy.update_metadata broken in 2.0.3

### ISSUE THREAD

**Issue by Calypso971** [savonet/ocaml-cry#8]
After upgrading from 2.0.2 to 2.0.3 I noticed that my dynamic metadata wasn't updated anymore. After some investigation I was able to pinpoint it to the `icy.update_metadata` call that apparently is broken in 2.0.3. After switching liquidsoap back to 2.0.2 again it worked without a problem.

Testscript:
```
radio = input.harbor("radio", id="radio", port=1234, user="source", password="hackme", buffer=5., max=20.,replay_metadata=false)

m = [("title", "This is - a test"),("url","http://google.com/image.png")]
icy.update_metadata(mount="/mystream",password="dummy",host="localhost",port=8000,m)

output.icecast(
  %mp3(bitrate=128, stereo=true,samplerate=44100,internal_quality=3),
  host="127.0.0.1",
  port=8000,
  public=false,
  fallible = true,
 description="My stream",
 mount="/mystream",
 icy_metadata="true",
 password = "dummy",
 url = "https://www.mystream.net",
 name = "Just listen",
 genre = "Pop",
 radio)
```
You'd expect that after running the stream has "This is - a test" as title metadata, it does this in 2.0.2 and previous versions, but not in 2.0.3

 - OS: Ubuntu 20.04 LTS (Focal)
 - Version: 2.0.3 (pre-compiled deb from github)

**Comment by Calypso971** [comment]
Great to hear you've found the cause - wasn't able to finish a testscript yet, and was planning to this evening. Now I've time left! Woohoo!

**Comment by toots** [comment]
This is now fixed with the `0.6.7` release of `ocaml-cry`. There's a `opam` package on the way. If you need a liquidsoap build with it, I'd recommend to try the `v2.0.4-preview` branch/CI assets. Let me know if you need more info. Thanks for reporting!
